spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -290,6 +290,12 @@
       "description": "MBeanServer bean name.",
       "defaultValue": "mbeanServer"
     },
+    {
+      "name": "spring.jmx.unique-names",
+      "type": "java.lang.Boolean",
+      "description": "Wether if unique runtime object names should be ensured.",
```

Comment:
Let's reuse the existing doc "Whether to ensure that ObjectNames are modified in case of conflict."